### ISSUE TITLE
alt attribute

### ISSUE THREAD

**Issue by AnthelmeLC** [advanced-cropper/vue-advanced-cropper#234]
Hi !

Is it possible to add alt attribute to your component, and set it to the img element built to improve the accessibility please ?

**Comment by AnthelmeLC** [comment]
Yes please :)

**Comment by Norserium** [comment]
@AnthelmeLC, and what it would give to an user? The cropper still won't be accessible.

**Comment by AnthelmeLC** [comment]
it add a description for browser reader. And yes the component will not be 100% accessible, but i'm not an expert, and just try to contribute like i can. 1% accessible is better than 0% and it's a good start to improve your accessibility .

**Comment by Norserium** [comment]
@AnthelmeLC, there is truth in your words. But this approach is opposite to system approach, that I prefer. 

Did you see any cropper that support accessibility? It would be helpful to me to understand the whole approach that should be implemented.

**Comment by AnthelmeLC** [comment]
Sorry for the long time i didn't answered.

I only use your cropper, so i can't help more sorry.
